src/main/java/com/nvidia/spark/ml/linalg/JniRAPIDSML.java

```diff
@@ -65,7 +65,7 @@ public static JniRAPIDSML getInstance() {
 
   public native void dgemm(int transa, int transb, int m, int n, int k, double alpha, double[] A, int lda, double[] B,
                            int ldb, double beta, double[] C, int ldc, int deviceID);
-  public native void dgemm_b(int m, int n, int k, double[] A, double[] B, double[] C, int deviceID);
-
+  public native long dgemmWithColumnViewPointer(int transa, int transb, int m, int n, int k, double alpha, double[] A,
```

Comment:
Since this is suppose to be an example, it would probably help to have some Javadoc for this native method explaining what the parameters are.  For example this just says `long B` which may confuse readers who don't realize `B` is the address of a `cudf::column_view *` that represents the B data (transposed).

Reply:
Updated.

Reply:
Still looks the same,  was a commit not pushed?